addons/crm/tests/test_crm_pls.py

```diff
@@ -207,11 +207,7 @@ def test_predictive_lead_scoring(self):
         lead_13_no_team_proba = leads[13].automated_probability
         self.assertTrue(lead_13_team_3_proba != leads[13].automated_probability, "Probability for leads with no team should be different than if they where in their own team.")
         # Todo: Make this test fully independent from demo data
-        if not loaded_demo_data(self.env):
-            expected_proba = 35.19
-        else:
-            expected_proba = 36.65
-        self.assertAlmostEqual(lead_13_no_team_proba, expected_proba, places=2)
+        self.assertAlmostEqual(lead_13_no_team_proba, 35.58, places=2)
```

Comment:
Are the probabilities similar with and without the demo data?

- If they aren't similar, can we keep the `if`/`else` statement and only update the expected probabilities when there is demo-data?
- If they are similar, you can green the task :smiley:

Reply:
Hello @julien-banken 
Okay after some testing we have to keep the if-else statement.
Thanks for the help :blush: